addons/gdUnit4/src/core/GdUnitSceneRunnerImpl.gd

```diff
@@ -41,11 +41,11 @@ func _init(p_scene, p_verbose :bool, p_hide_push_errors = false):
 	set_time_factor(1)
 	# handle scene loading by resource path
 	if typeof(p_scene) == TYPE_STRING:
-		if !FileAccess.file_exists(p_scene):
+		if !ResourceLoader.exists(p_scene):
```

Comment:
I hope this change is downward compatible to older Godot versions, let run us the CI tests ;)